src/role-access/services/allocate-role.service.ts

```diff
@@ -12,6 +12,7 @@ export class AllocateRoleService {
   public static allocateRoleBaseUrl = '/api/role-access/allocate-role';
   public static roleUrl = '/api/role-access/roles';
   public static accessManagementUrl = '/api/am';
+  public static specificAccessUrl = '/api/specific-access-request/request-more-information';
```

Comment:
Specific access url should be separate from 'request-more-information' which can be appended to the string